deploy/application/files/values.yaml.tpl

```diff
@@ -1,12 +1,9 @@
-# https://github.com/DND-IT/app-helm-chart/blob/4.1.0/values.yaml
+# https://github.com/DND-IT/helm-charts/blob/main/charts/app/values.yaml
```

Comment:
@DCamma [minor] Link to the current included version to make sure you're checking the values of the same version.